internal/resources/ekscluster/helpers_test.go

```diff
@@ -582,12 +582,120 @@ func TestClusterSpecEqual(t *testing.T) {
 			},
 			result: true,
 		},
+		{
+			name: "addon configs are set equal",
```

Comment:
I don't see a set equality test case. They both look identically ordered to me

Reply:
@asutoshpalai  equality test works now and I have pushed the changes.